SshAgentLibTests/PpkFormatterTest.cs

```diff
@@ -429,6 +429,74 @@ public void PpkFileParseDataTest()
         Assert.That(target.Comment, Is.EqualTo(comments[i]));
       }
     }
+
+    [Test]
+    public void ParsePpkv3()
+    {
+      ISshKey target;
+
+      PpkFormatter.WarnOldFileFormatCallback warnOldFileNotExpected = delegate()
```

Comment:
To be consistent, we can use `=>` everywhere instead of `delegate`.

Reply:
I replaced delegate() with ()=>